spiffworkflow-backend/src/spiffworkflow_backend/services/spec_file_service.py

```diff
@@ -481,7 +487,7 @@ def check_for_message_models(
                         ):
                             raise ValidationException(
                                 "Message model is already used to start process model"
-                                f"'{process_model_info.process_group_id}/{process_model_info.id}'"
+                                f"'{process_model_info.process_group}/{process_model_info.id}'"
```

Comment:
this should probably become just `process_model_info.id` instead of the f string